packages/ventoy-bin/ventoy-bin.pacscript

```diff
@@ -0,0 +1,90 @@
+pkgname=ventoy-bin
+pkgver=1.0.97
+pkgrel=1
+pkgdesc="A new bootable USB solution"
+maintainer="xdavius <xdavius@github.com>"
+arch=('amd64')
+url="http://www.ventoy.net"
+license=('GPL-3.0-or-later')
+depends=('bash' 'util-linux' 'xz-utils' 'dosfstools')
+optdepends=('libgtk-3-dev: GTK3 GUI'
+  'qtbase-5-dev: Qt5 GUI'
+  'polkit: run GUI from application menu')
+provides=("${pkgname%-bin}")
+conflicts=("${pkgname%-bin}")
+install="${pkgname%-bin}.install"
+source=("https://github.com/ventoy/Ventoy/releases/download/v${pkgver}/${pkgname%-bin}-${pkgver}-linux.tar.gz"
+  "https://aur.archlinux.org/cgit/aur.git/plain/sanitize.patch?h=ventoy-bin"
+  "https://aur.archlinux.org/cgit/aur.git/plain/ventoy?h=ventoy-bin"
+  "https://aur.archlinux.org/cgit/aur.git/plain/ventoy-extend-persistent?h=ventoy-bin"
+  "https://aur.archlinux.org/cgit/aur.git/plain/ventoygui?h=ventoy-bin"
+  "https://aur.archlinux.org/cgit/aur.git/plain/ventoy-persistent?h=ventoy-bin"
+  "https://aur.archlinux.org/cgit/aur.git/plain/ventoyplugson?h=ventoy-bin"
+  "https://aur.archlinux.org/cgit/aur.git/plain/ventoy.desktop?h=ventoy-bin"
+  "https://aur.archlinux.org/cgit/aur.git/plain/ventoyweb?h=ventoy-bin")
+sha256sums=('1368a9082c9db25958e5407ef4984322d90d3bb00905337d4bd41678e06c1150'
+  'SKIP'
+  'SKIP'
+  'SKIP'
+  'SKIP'
+  'SKIP'
+  'SKIP'
+  'SKIP'
+  'SKIP')
+
+prepare() {
+  cd "${_archive}"
+  export CARCH="x86_64"
```

Comment:
Replace occurrences of `CARCH` with `x86_64`. There's only one `arch` anyway so it's redundant
```suggestion
```

Reply:
> Replace occurrences of `CARCH` with `x86_64`. There's only one `arch` anyway so it's redundant

ok